libsolidity/interface/StandardCompiler.cpp

```diff
@@ -146,6 +146,47 @@ Json::Value collectEVMObject(eth::LinkerObject const& _object, string const* _so
 	return output;
 }
 
+bool isTargetRequired(Json::Value const& _targets, string const& _target)
+{
+	for (auto const& target: _targets)
+		if (target == _target)
```

Comment:
Need to support sub matches, e.g. `evm.assembly` should be enabled by both `evm.assembly`, `evm`, `evm.*`, etc.